src/lib/octokit.js

```diff
@@ -8,11 +8,15 @@ const RETRY_LIMIT = 5
 
 let client
 
+const labels = []
```

Comment:
**suggestion**: a `Set` would be a better fit for this, it would prevent duplicate labels, and would be easier to clear in `clearPendingMessages`